tests/sentry/runner/commands/test_configoptions.py

```diff
@@ -92,33 +85,41 @@ def assert_output(rv):
 
             # The script produces log lines when DRIFT is detected. This
             # makes it easier to surface these as Sentry errors.
-            # This also means the output is polluted with a log line
-            # because reconfiguring the logger in the test is quite tricky
-            # as it is initialized at the beginning of the test.
-            # So we just split the output in two and check each part
-            # independently.
-            output_before_log = "\n".join(
+
+            set_output = "\n".join(
                 [
-                    SET_MSG % ("int_option", 40),
-                    UPDATE_MSG % ("str_option", "old value", "new value"),
-                    SET_MSG % ("map_option", {"a": 1, "b": 2}),
-                    SET_MSG % ("list_option", [1, 2]),
-                    DRIFT_MSG % "drifted_option",
+                    ConsolePresenter.SET_MSG % ("int_option", 40),
+                    ConsolePresenter.SET_MSG % ("map_option", {"a": 1, "b": 2}),
+                    ConsolePresenter.SET_MSG % ("list_option", [1, 2]),
                 ]
             )
-
-            output_after_log = "\n".join(
+            drift_output = "\n".join(
                 [
-                    DB_VALUE % "drifted_option",
+                    ConsolePresenter.DRIFT_MSG % "drifted_option",
+                    ConsolePresenter.DB_VALUE % "drifted_option",
                     "- 1",
                     "- 2",
                     "- 3",
                     "",
-                    CHANNEL_UPDATE_MSG % "change_channel_option",
                 ]
             )
-            assert output_before_log in rv.output
-            assert output_after_log in rv.output
+
+            channel_update_output = "\n".join(
+                [
+                    ConsolePresenter.CHANNEL_UPDATE_MSG % "change_channel_option",
+                ]
+            )
+
+            update_output = "\n".join(
+                [
+                    ConsolePresenter.UPDATE_MSG % ("str_option", "old value", "new value"),
+                ]
+            )
+
+            assert drift_output in rv.output
+            assert set_output in rv.output
+            assert channel_update_output in rv.output
+            assert update_output in rv.output
```

Comment:
This is different than before. 
Is there any reason we cannot assert the messages are in the right sequence? 
Before we broke the assertion in two because there was a logger message in between the two chunks and that was really hard to assert on. 
But ideally we want to assert on the whole chunk of output to ensure no additional options are updated.